src/libs/AccountUtils.ts

```diff
@@ -5,4 +5,6 @@ import type {Account} from '@src/types/onyx';
 const isValidateCodeFormSubmitting = (account: OnyxEntry<Account>) =>
     !!account?.isLoading && account.loadingForm === (account.requiresTwoFactorAuth ? CONST.FORMS.VALIDATE_TFA_CODE_FORM : CONST.FORMS.VALIDATE_CODE_FORM);
 
-export default {isValidateCodeFormSubmitting};
+const isAccountIDOddNumber = (accountID: number) => accountID % 2 === 1;
```

Comment:
Can we please move this to the `permissions` file so that it is treated just like any of the other betas?

Reply:
Fair, I think we were not looking at this as a beta, but I can see where you are coming from. @koko57 do you think you could move this around to permissions file?